.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xbmc/addons/kodi-addon-dev-kit"]
+	path = xbmc/addons/kodi-addon-dev-kit
+	url = git://github.com/AlwinEsch/kodi-addon-dev-kit.git
```

Comment:
OK, here comes a hard part from me. It is to use the `kodi-addon-dev-kit` as part inside kodi itself and as independent repository for add-on developers.

Has searched about ways to let it inside Kodi itself and to perform a git download with a result as indepenendent code like on https://github.com/AlwinEsch/kodi-addon-dev-kit, but nothing found.

Anyone a idea to have it easy alone for developers but still full inside Kodi?

On travis it worked as own repository for add-ons (https://travis-ci.org/xbmc/xbmc/builds/121236644)